### ISSUE TITLE
Links not working in 0.26 targeting .NetFramework 4.7

### ISSUE THREAD

**Issue by raulcristiann** [xoofx/markdig#574]
Looks like in version 0.23 everything works fine but starting with 0.24 links are not working anymore.
Try this code:

```
using System;
using Markdig;

public class Program
{
    public static void Main()
    {
        var pipeline = new MarkdownPipelineBuilder()
               .UseAdvancedExtensions()
               .Build();

        var notWorking = @"
text ==Marked text== text.

this is ![](https://pbs.twimg.com/profile_images/234/kt0hiSmv_400x400.jpg) image

> Dorothy followed her through many of the beautiful rooms in her castle.

Content with **bold** more content

Content with  ***italic bold***  more content

this [link1] sparta

[link1]: /url ""title1""

    internal class Sparta
        {
            public void Say()
            {
                Console.WriteLine(""This is Sparta"");
            }
        }

<div>
<span style = ""font-weight:bold;"" >
this is
a test
this should be
visible
</span>

</div>

<div> <span> asdasdasd asdads</span> </div>
";


        var working = @"

this [link1] sparta

[link1]: \url ""title1""";
        var workingResult = Markdown.ToHtml(working, pipeline);
        var notWorkingResult = Markdown.ToHtml(notWorking, pipeline);

        Console.WriteLine(notWorkingResult);
        Console.WriteLine(workingResult); //Check links
    }
}
```

**Comment by raulcristiann** [comment]
Any updates on this?

**Comment by MihaZupan** [comment]
Looks like it happens when a `LinkReferenceDefinition` containing a title is followed by a new line.

```c#
string markdown = @"
[foo]

[foo]: /url ""bar""
";
```
vs
```c#
string markdown = @"
[foo]

[foo]: /url ""bar""";
```

cc: @generateui

**Comment by xoofx** [comment]
It looks like it is fixed in 0.28.1
